components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksInterceptor.kt

```diff
@@ -146,6 +147,7 @@ class AppLinksInterceptor(
         return when {
             url == null -> return null
             url.startsWith(WWW) -> url.replaceFirst(WWW, "")
+            url.startsWith(M) -> url.replaceFirst(M, "")
```

Comment:
@psymoon also landed code recently that checks for mobile URls. I think soon we'll want a share utility / extension function? https://github.com/mozilla-mobile/android-components/blob/master/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt#L275

Reply:
That's great.  I'll keep an eye out for the shared utility / extension function.

Reply:
Like you mentioned offline that there's not a lot of duplication right now.  I'll keep this in mind if I make anymore changes in the future relating to mobile URLs.  Right now I agree this change is not introducing too much duplication.  I'll merge it.  Thanks,